docs/courseInstance.md

```diff
@@ -52,7 +52,7 @@ This file specifies basic information about the course instance:
 ??? note "Format specification for `infoCourseInstance.json`"
 
     ```json
-    --8<-- "apps/prairielearn/src/schemas/schemas/infoCourseInstance.json"
+    --8<-- "apps/prairielearn/src/schemas/infoCourseInstance.ts"
```

Comment:
I'm not convinced that showing the Zod schema in the docs is useful. IMO it is in fact much less human-readable than the old JSON schema.

Reply:
I'm fine removing it from the documentation, unless you have an alternative you want to replace it with?

Reply:
I removed the mention in 2186244dc6d4a51708c7a1344f459e4c88ab6145, but I am open to changes / modifications / reverts.

Reply:
I think omitting it is the right move for now.